src/btree/bt_read.c

```diff
@@ -232,9 +255,10 @@ __las_page_instantiate(WT_SESSION_IMPL *session, WT_REF *ref)
     cache->las_reader = true;
     __wt_readlock(session, &cache->las_sweepwalk_lock);
     cache->las_reader = false;
+    incr = total_incr = 0;
```

Comment:
`__create_birthmark_upd()` calls `__wt_update_alloc()`, and `__wt_update_alloc()` should be clearing `*sizep` out of sheer paranoia.

Reply:
Without `incr = 0` compiler is generating warnings as uninitialized variable usage even if it is not used as uninitialized. so I added the initialization.